app/controllers/event_notes_controller.rb

```diff
@@ -0,0 +1,82 @@
+class EventNotesController < ApplicationController
+  include SerializeDataHelper
+
+  rescue_from Exceptions::EducatorNotAuthorized, with: :redirect_unauthorized!
+
+  before_action :authorize!, except: [:names]
+
+  def authorize!
+    student = Student.find(params[:student_id])
+    educator = current_educator
+    raise Exceptions::EducatorNotAuthorized unless educator.is_authorized_for_student(student)
+  end
+
+  def create
+    clean_params = params.require(:event_note).permit(*[
```

Comment:
This works and is easy to read but it would be a bit more idiomatic rails to enapsulate your permissible parameters in a private method. http://edgeguides.rubyonrails.org/action_controller_overview.html#strong-parameters